src/build_info.cpp

```diff
@@ -550,6 +550,10 @@ list_formatter video_settings_report_internal(const std::string& heading = "")
 		placeholder = "Running in non-interactive mode.";
 	}
 
+	if(!video::has_window()) {
+		placeholder = "Video not initialized yet.";
```

Comment:
"yet" implies that it will be done later, seems incorrect for a code path that's only for `--report`.

Reply:
Is this really that important? :\

Reply:
I guess not.